go-controller/pkg/ovn/base_network_controller.go

```diff
@@ -245,6 +253,23 @@ func (oc *BaseNetworkController) reconcile(netInfo util.NetInfo, setNodeFailed f
 		}
 	}
 
+	for _, ns := range reconcileNamespaces.List() {
```

Comment:
This method is tarting to get too big. One neat idea would be to have a separate method
```
// doReconcile performs the reconciliation after the controller NetInfo has already being
// updated with the changes. What needs to be reconciled should already be known and 
// provided on the arguments of the method. This method returns no error and logs them
// instead since once the controller NetInfo has been updated there is no point in retrying.
func (oc *BaseNetworkController) doReconcile(reconcileRoutes, reconcilePendingPods bool, reconcileNodes, reconcileNamespaces []string) {
    ....
}
```
including everything under line 232

Reply:
ok, done.